packages/neuron-wallet/src/models/keys/address.ts

```diff
@@ -15,8 +15,8 @@ export const publicKeyToAddress = (publicKey: string, isMainnet = false) => {
   const pubkey = publicKey.startsWith('0x') ? publicKey : `0x${publicKey}`
   return scriptToAddress(
     {
-      codeHash: systemScripts.SECP256K1_BLAKE160.CODE_HASH,
-      hashType: systemScripts.SECP256K1_BLAKE160.HASH_TYPE,
+      codeHash: systemScriptsMainnet.SECP256K1_BLAKE160.CODE_HASH,
```

Comment:
Since `codeHash` and `args` of `SECP256K1_BLAKE160` are the same for both mainnet and testnet, it's more simple to just use systemScripts**Mainnet** here

Reply:
It's correct from the perspective of a result, but incorrect from the perspective of logic/progress. So it's bug-prone. What if `secp256k1` lock is updated on testnet first?

Reply:
how about this, calling `const SCRIPTS = isMainnet ? predefined.LINA.SCRIPTS : predefined.AGGRON4.SCRIPTS` is very anoying, which is what I did in very first trial of refactoring `@nervosnetwork/ckb-sdk-js`, and in some util functions it's hard to get the `isMainnet` state, otherwise the original signature of util functions may change by adding a input `isMainnet` parameter.

Here is what I plan: refactor `systemScripts` by adding a `systemScripts.ts` file in utils folder, but make the fields of it the same as `@nervosnetwork/ckb-sdk-js`, and it's based ont the lumos config. If anything changes in system scripts, we just need to change the `systemScripts.ts` file.

updated by: [4b5e45d](https://github.com/nervosnetwork/neuron/pull/2814/commits/4b5e45dd5085168681147740f52dad59020b5019)